stratum/hal/lib/barefoot/bf.proto

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020-present Open Networking Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+option cc_generic_services = false;
+
+package stratum.hal.barefoot;
+
+import "p4/config/v1/p4info.proto";
+
+// This message describes the public Barefoot specific definition of a single P4
+// program. It is passed to Stratum as the p4 device config of the P4Runtime
+// ForwardingPipelineConfig message.
+message BfPipelineConfig {
+  message Profile {
+    string profile_name = 1;
+    bytes context = 2;
+    bytes binary = 3;
+    repeated uint32 pipe_scope = 4;
+  }
+  string p4_name = 1;
+  bytes bfruntime_info = 2;
+  repeated Profile profiles = 3;
+}
```

Comment:
This message is now 100% identical to `ForwardingPipelineConfig` in bfruntime.proto.